controllers/etcdcluster_controller_test.go

```diff
@@ -279,8 +283,8 @@ func (s *controllerSuite) testClusterController(t *testing.T) {
 				if err != nil {
 					return err
 				}
-				if len(peers.Items) != 3 {
-					return fmt.Errorf("wrong number of peers: %#v", len(peers.Items))
+				if len(peers.Items) != expectedReplicas+1 {
+					return fmt.Errorf("wrong number of peers. expected: %d, actual: %d", expectedReplicas, len(peers.Items))
```

Comment:
Why the `expectedReplicas+1` here? Your example error message is strange too:

```
wrong number of peers. expected: 3, actual: 3
```

Leftover debug code?

Reply:
Oops. That was how I triggered the error essage....didn't mean to commit it....I'll fix that now...